### PR TITLE
Re-emphasize JupyterHub

### PR DESCRIPTION
**Pull request by yuvipanda**
Our current primary product is *customized JupyterHubs*,
so let's lead with that. The secondary section has a little
bit more explanation & context.

Trimmed so that the title is < 2 lines, and the description
is also < 3 lines.

Emphasis is on the customized JupyterHub part, since that's
what we really want to communicate to our users.

Before:

image

After:

image

Note that it's much easier to see what we can actually do for people right now this way.

### CODE REVIEW

**Comment by yuvipanda** [comment]
The more I think about it, our current value proposition is pretty much just 'customized jupyterhubs', so we should emphasize that. But we aren't selling *that*, so we need to make sure we don't appear as just a service provider

File: content/home/hero.md [hunk 1]
```diff
@@ -47,4 +47,4 @@ hero_media = "2i2c-hub-overview.png"
 #  label = '2i2c provides open-source infrastructure for interactive computing at institutions of research and education.'
 +++
 
-2i2c is a non-profit organization that runs and supports customized JupyterHubs for educational & research users.
+2i2c is a non-profit organization that develops and runs fully open source interactive computing infrastructure for your community.
```
Comment (choldgraf) [inline content/home/hero.md:50]:
```suggestion
2i2c is a non-profit organization that develops and runs open source interactive computing infrastructure for your community.
```

**Comment by choldgraf** [comment]
A question that is growing in my mind - how much of 2i2c.org should be about the Hub Service, vs. about 2i2c in general?

I feel like we are currently defining everything 2i2c does as "The Managed JupyterHub Service". That is sort-of true right now (though, not quite, we also run development projects in JupyterHub that aren't related to the Hub Service, like helping with Jupyter Meets the Earth, or doing Jupyter Book work). Is that work being lost in the current focus on "Running customized JupyterHubs"?

**Comment by yuvipanda** [comment]
I think of 2i2c as doing two things interesting to others right now:

1. Supporting open source work (possibly funded)
2. Running hubs for (and with) people

I think we can separate and section them into the home page in a way that clearly explains what we're doing and why we're doing it. What gets top billing can be changed easily over time, but managed hubs maybe what it begins with.

**Comment by choldgraf** [comment]
I think that's a reasonable breakdown, though I'd add one other thing: we perform focused development for use-cases that benefit research and education. The example I'd give here is Jupyter Book. This goes beyond general "open source support" - it is focusing attention and resources on a complex project that has implications for the scholarly community. I think we should highlight our leadership in these spaces in a way that goes beyond general "open source support". Does that make sense?

That said, I agree with your assessment that we can change this kind of thing easily, I mostly wanna bring this up to see if you agree with the framing of this as an "extra third thing".

**Comment by yuvipanda** [comment]
I've been working on some restructuring, and writing is hard. Here's the home page I'm trying to work towards:

|  |
| - |
| Hero / Lead (as in the screenshot in this PR) |
| Call out to managed jupyterhubs |
| JupyterHub use cases - 4 kinds of hubs |
| JupyterHub features or Testimonials (but not both) |
| Call out to Open Source work |
| Describe grants we have (CZI, Ryan's Moore, JupyterBook) |
| Describe other projects we are part of |
| Team  & values |

This should also let us link out to more detailed pages for our infrastructure and open source work.

How do you feel about separating this from the rewording just in this PR already though?

**Comment by yuvipanda** [comment]
Had a chat with @choldgraf earlier, and here is where we settled on for now:

1. Home page's goal is to express our mission, and link out to our prominent work: Managed JupyterHubs, and various 'projects'. Pretty much nothing else - take inspiration from openai.com
2. Managed JupyterHubs page will become our 'landing' page for people interested in that. This lets us pack more information with sharp short prose without needing to be vague.
3. Focus on 'projects' that we are focused on, rather than OSS projects we contribute to in general. So this would be things from the CZI grant, JupyterBook (via the grant), the Pangeo hire, etc
4. Make a 'values' page that's a top level item, focused on explaining our values - including content from the current 'open source' page.
5. Get testimonials from people who have used our managed service page for social cred.
6. Make the home page's hero section less crowded (see #51)

Does that cover everything, @choldgraf?

**Comment by yuvipanda** [comment]
The actual contents of this PR won't be used, so closing it.

**Pull request closed without merging**